### ISSUE TITLE
domLayout: 'autoHeight' broken in IE11

### ISSUE THREAD

**Issue by jgood044** [ag-grid/ag-grid#2968]
**I'm submitting a ...**
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```

When setting `domLayout = 'autoHeight'` in IE11, the grid height is not rendered correctly. Only the header and scrollbar are shown and no rows are visible.

**Providing a Reproducible Scenario**

- Set domLayout to 'autoHeight' in IE11
- Observe issue with grid height

*Not working in official [ag-grid demo](https://www.ag-grid.com/javascript-grid-width-and-height/#example-auto-height) on the grid size documentation page.

[Plunker](https://plnkr.co/edit/rrDHcUPPAeufUp4JtSNC?p=preview)

**Current behavior**
The grid does not render the correct height when `domLayout = 'autoHeight'`

**Expected behavior**
The grid renders the correct height automatically.
* **ag-Grid version:** 20.0.0

* **Browser:** IE11

* **Language:** All

**Comment by jgood044** [comment]
Appears to be fixed with release [20.1.0](https://www.ag-grid.com/ag-grid-changelog/?fixVersion=20.1.0)

**Comment by jgood044** [comment]
Appears to now be fixed with release [20.1.0](https://www.ag-grid.com/ag-grid-changelog/?fixVersion=20.1.0)
